### ISSUE TITLE
GA RPN

### ISSUE THREAD

**Issue by sehyungp92** [open-mmlab/mmdetection#3144]
I had a quick question on the Guided-Anchoring RPN. The paper suggests that there were notable gains from discarding the RPN used during training and replacing with pre-computed GA-RPN proposals to finetune (and also used for inference):

- Does this involve training the data using a GA RPN config file then importing the proposals to, for instance, a Mask R-CNN, or does this mean replacing the RPN with a pre-trained GA-RPN for finetuning?
- Either way, how exactly do you import proposals to a Mask R-CNN model / replace the RPN with a pre-trained GA-RPN before finetuning?

Thanks.

**Comment by hellock** [comment]
Using the proposals generated offline by GA-RPN.

**Comment by sehyungp92** [comment]
Many thanks for your response.

Can you kindly let me know how I can import proposals generated offline by GA-RPN to a Mask R-CNN model for finetuning.

**Comment by hellock** [comment]
Train the Fast-R-CNN like models.

**Comment by sehyungp92** [comment]
Thanks for your advice.

Does this mean that it is not straightforward to use the finetuning method with, for instance, Mask R-CNN or HTC?
